### PR TITLE
feat: only expose request and responses from api

### PR DESCRIPTION
**Pull request by zatlite**
# Summary
[DX-1251]

Tested working

```
core-sdk % ./node_modules/.bin/cucumber-js -t @tradeWithFee
Feature: Trading # ../../features/trading.feature:1

  @trading @ethSignature @tradeWithFee
  Scenario: Trading with fees # ../../features/trading.feature:21
    Given A new Eth wallet "maker"
    And A new Eth wallet "taker"
    And "maker" is registered
    And "taker" is registered
    And randomly L2 mint to "maker" of "minted"
    And NFT "minted" should be available through api
check nft: 1969653554 on address 0xB678442687254bea98DcB9F4F0eBbe1Ef293c580
...    And banker transfer "0.000012" eth to "taker"
    When "maker" creates sell order "sellOrder" of "minted" NFT for sell for "0.00001" eth
    And "taker" fills order "sellOrder" to create trade "trade1" with fees
    Then trade "trade1" should be available through api
check trade: 25476
...    And api should show that "taker" owns the NFT "minted"
    And "maker" transfer "0.00001" eth to banker

1 scenario (1 passed)
12 steps (12 passed)
```
# Why the changes

# Things worth calling out

[DX-1251]: https://immutable.atlassian.net/browse/DX-1251?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/exportApis.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  GetSignableCancelOrderRequest,
+  TradesApiGetTradeRequest,
+  TradesApiListTradesRequest,
+  GetSignableTradeRequest,
+  TokensApiGetTokenRequest,
+  TokensApiListTokensRequest,
+  TransfersApiGetTransferRequest,
+  TransfersApiListTransfersRequest,
+} from './api';
+
```
Comment (matthewmuscat) [inline src/exportApis.ts:32]: How about configuration? https://github.com/immutable/imx-core-sdk/blob/v1.0.0-beta.1/src/api/index.ts#L17

Comment (Kastet) [reply src/exportApis.ts:32]: Do we need to? We provide constructors e.g. `Config.SANDBOX`

Comment (matthewmuscat) [reply src/exportApis.ts:32]: Not sure, but we're [exporting them still. ](https://github.com/immutable/imx-core-sdk/blob/v1.0.0-beta.1/src/api/configuration.ts#L26) Why are we exporting them if we aren't exposing them?

File: src/index.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
 export * as Contracts from './exportContracts';
 export * from './exportUtils';
+export * from './exportApis';
```
Comment (Kastet) [inline src/index.ts:3]: Maybe `exportApis` -> `exportApiModels`?

Comment (zatlite) [reply src/index.ts]: [5a30b22](https://github.com/immutable/imx-core-sdk/pull/183/commits/5a30b223f8651d057d28cde506d6b64bdea26fdc)

File: src/exportApis.ts (hunk 1 above)
Comment (matthewmuscat) [reply src/exportApis.ts]: Not needed as it's autogenerated and. Discussed offline.

**Review by matthewmuscat — APPROVED** [review]
looks good!

**Review by Kastet — APPROVED** [review]
👍

**Pull request merged**